src/sourcemaps/sourcemaps-wizard.ts

```diff
@@ -16,14 +16,19 @@ import { SourceMapUploadToolConfigurationOptions } from './tools/types';
 import { configureVitePlugin } from './tools/vite';
 import { configureSentryCLI } from './tools/sentry-cli';
 import { configureWebPackPlugin } from './tools/webpack';
+import { configureTscSourcemapGenerationFlow } from './tools/tsc';
 
 interface SourceMapsWizardOptions {
   promoCode?: string;
 }
 
-type SupportedBundlers = 'webpack' | 'vite' | 'rollup' | 'esbuild';
-type SupportedTools = 'sentry-cli';
-type SupportedBundlersTools = SupportedBundlers | SupportedTools;
+type SupportedTools =
```

Comment:
good cleanup, i initially separated these because I was toying with the idea of creating one flow for the 4 bundlers which is why i split it up. But now that we have separate flows (which is probably better anyway) it was just unnecessary 😅